src/gsy_e/gsy_e_core/rq_job_handler.py

```diff
@@ -26,19 +26,21 @@ def decompress_and_decode_queued_strings(queued_string: bytes) -> Dict:
 
 def launch_simulation_from_rq_job(scenario: bytes, settings: Optional[Dict],
                                   events: Optional[str], aggregator_device_mapping: str,
-                                  saved_state: bytes, job_id: str):
+                                  saved_state: bytes, job_id: str,
+                                  connect_to_profiles_db: bool = True):
     # pylint: disable=too-many-arguments, too-many-locals
     """Launch simulation from rq job."""
     logging.getLogger().setLevel(logging.ERROR)
     scenario = decompress_and_decode_queued_strings(scenario)
-    gsy_e.constants.CONFIGURATION_ID = scenario.pop("configuration_uuid")
-    if "collaboration_uuid" in scenario:
-        gsy_e.constants.EXTERNAL_CONNECTION_WEB = True
-        GlobalConfig.IS_CANARY_NETWORK = scenario.pop("is_canary_network", False)
-        gsy_e.constants.RUN_IN_REALTIME = GlobalConfig.IS_CANARY_NETWORK
+    if isinstance(scenario, dict):
```

Comment:
Can you please explain why this is needed? `scenario` should be a dict after `decompress_and_decode_queued_strings`

Reply:
This is because a scenario can be `None`, a dict or a string regarding [this](https://github.com/gridsingularity/gsy-e/blob/6217ead527f630fb2082a5dc1bb0a9a9b180babd/src/gsy_e/gsy_e_core/rq_job_handler.py#L116-L122) part of the code.

Reply:
Thanks! Understood. I think we can remove the functionality of being able to pass a scenario name at some point. Nobody is using this. Maybe worth another hackathon ticket!?

Reply:
@hannesdiedrich 
https://gridsingularity.atlassian.net/browse/GSYE-339